pytket/docs/changelog.rst

```diff
@@ -12,6 +12,7 @@ Minor new features:
   that can be rendered by the ``graphviz`` package.
 * New pass and transform `DecomposeTK2` to decompose TK2 gates using the
   approximate KAK decomposition.
+* ``GlobalisePhasedX`` uses fewer Rz rotations.
```

Comment:
Maybe clarify that this is a compiler pass?

Reply:
Done!